workspaces/tech-insights/plugins/tech-insights-client/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@backstage-community/plugin-tech-insights-client",
```

Comment:
```suggestion
  "name": "@backstage-community/plugin-tech-insights-node",
```

The naming convention would be to have this be `node` and not `client`

Reply:
I have been assuming it should be client as this the naming convention the OpenAPI tooling generates and therefore the ecosystem move in that direction. 

Is this assumption incorrect? 

Also the client would be imported in the frontend too, so the node declaration would not be correct.

Reply:
It's based on the role and you have it as `node-library` so that's why I suggested `node`. News to be that the OpenAPI tooling generates it that way. That's not a change I've seen documented anywhere so we should probably come to an agreement on that.

For reference the roles are here: https://backstage.io/docs/tooling/cli/build-system#package-roles

Reply:
@Rugvip @aramissennyeydd, what's your thoughts on this? I'm fine with `client` but that's not what I've seen done before which is what drove my initial comment.

Reply:
It's a little up in the air, the standard I think we're trying to end up with is `-common` and having a `/client` entry point so you could just `import {...} from '@backstage-community/plugin-tech-insights-common/client'`. From Rugvip's comment [here](https://github.com/backstage/backstage/pull/23610#discussion_r1546376919).

Reply:
For reference: https://backstage.io/docs/openapi/01-getting-started#generating-a-typed-client-from-a-spec

Reply:
Have been waiting for the above PR to be merged to update those docs 😞 , either is fine for the time being.

Reply:
Let's go with `common` and make sure to update the role to `common-library`, please.  👍